server/pbenchinacan/container-build.sh

```diff
@@ -13,6 +13,9 @@ PB_SERVER_IMAGE_TAG=${PB_SERVER_IMAGE_TAG:-$(< ${GITTOP}/jenkins/branch.name)}
 RPM_PATH=${RPM_PATH:-/root/sandbox/rpmbuild/RPMS/noarch/pbench-server-*.rpm}
 KEYCLOAK_CLIENT_SECRET=${KEYCLOAK_CLIENT_SECRET:-"client-secret"}
 
+# Default target registry to use.
+PB_CONTAINER_REG=${PB_CONTAINER_REG:-$(<${HOME}/.config/pbench/ci_registry.name)}
+
 # Locations on the host
 GITTOP=${GITTOP:-$(git rev-parse --show-toplevel)}
```

Comment:
It looks like one of my recent PR's broke something:  the definition of `GITTOP` here is now referenced in a _preceding_ line (line 12).

@portante, is that something you would be amenable to addressing (e.g., in this PR), or should I post a new one?

Reply:
Fixed.

Reply:
Thanks!  It's interesting that that didn't show up before now.

BTW, I hit another interesting foible:
```
$ RPM_PATH=[...] PB_CONTAINER_REG=[...] ./server/pbenchinacan/container-build.sh
/usr/bin/env: ‘bash -e’: No such file or directory
/usr/bin/env: use -[v]S to pass options in shebang lines
```

So, when the script shebang is `#!/usr/bin/env bash` instead of `#!/bin/bash`, we need to specify the `-S` switch to `env` if we specify switches to `bash`.

Any thoughts on which idiom we should use?  (I.e., `/usr/bin/env -S bash -e` vs. `/bin/bash -e`?)

Reply:
FWIW, the reason why these problems didn't show up before is that, in the CI pipeline, we've already defined `PB_SERVER_IMAGE_TAG` and we invoke this script with an explicit bash command (so that the sub-commands are echoed):
```
                sh 'bash -ex ./server/pbenchinacan/container-build.sh'
```
🥴

Reply:
Fixed.